content/zh/2022-03-25-integrating-skywalking-with-source-code/index.md

```diff
@@ -0,0 +1,56 @@
+---
+title: "将 Apache SkyWalking 与源代码集成"
+author: "Brandon Fergerson"
+date: 2022-03-25
+description: "在本文中，我们将介绍开源实时编码平台 Source++ 中的一个新概念，旨在让开发人员更直观地监控生产应用程序。"
+tags:
+- Observability
+---
+
```

Comment:
Let's add a statement, this is translated by which tool.

Reply:
I used Fiverr to avoid using google translate. Looks like they might have just used google translate anyway.

I've added a note that the original post is in English. Does that work?